pkg/component/registrycaches/registry_caches.go

```diff
@@ -391,14 +433,22 @@ source /entrypoint.sh /etc/distribution/config.yml
 								},
 							},
 						},
+						{
+							Name: registryCertsVolumeName,
+							VolumeSource: corev1.VolumeSource{
+								Secret: &corev1.SecretVolumeSource{
+									SecretName: secretTLS.Name,
```

Comment:
Looks like we mount TLS secrets with permissions `0640`. See https://github.com/gardener/gardener/pull/8790.

```suggestion
									SecretName:  tlsSecret.Name,
									DefaultMode: ptr.To[int32](0640),
```